### PR TITLE
feat(process-exporter): Add process-exporter to dev and metal

### PR DESCRIPTION
**Pull request by KaiyiLiu1234**
process-exporter is a useful tool for baremetal validation. It can be used for process to process performance metric comparison like cpu time.

### CODE REVIEW

**Comment by KaiyiLiu1234** [comment]
From testing, it appears fixing the remaining yamllint errors will not impact the process-exporter service, but I followed exactly what https://github.com/ncabatoff/process-exporter does and it violates the yamllint rules.

File: manifests/compose/validation/config/process-exporter.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+process_names:
+  - name: "{{.PID}}"
+    cmdline:
+    - '.+'
```
Comment (vprashar2929) [inline manifests/compose/validation/config/process-exporter.yaml:4]: same here
```
    cmdline:
      - ".+" # yamllint disable-line rule:quoted-strings

```

File: manifests/compose/validation/metal/compose.yaml [hunk 2]
```diff
@@ -143,11 +143,28 @@ services:
     networks:
       - libvirt-exporter-network
 
+  process-exporter:
+    image: ncabatoff/process-exporter
+    privileged: true
+    ports:
+      - 9256:9256
+    command:
+      - '--procfs=/host/proc'
```
Comment (vprashar2929) [inline manifests/compose/validation/metal/compose.yaml:152]: same here

**Comment by KaiyiLiu1234** [comment]
@vprashar2929 all addressed

**Review by sthaha — APPROVED** [review]

**Comment by vprashar2929** [comment]
@KaiyiLiu1234 Can you fix the commit message `clean(process-exporter): Add yamllint fixes` as `clean` is not a standard type as per conventional commit [specification](https://www.conventionalcommits.org/en/v1.0.0/) 

Also looks like there is one more yamllint fix required. Can you run `pip install pre-commit` and `pre-commit run --show-diff-on-failure --color=always --all-files` to fix that as well

**Comment by KaiyiLiu1234** [comment]
Command ran @vprashar2929

**Pull request merged**